apisix/plugin.lua

```diff
@@ -375,14 +375,15 @@ local function trace_plugins_info_for_debug(ctx, plugins)
     end
 end
 
-local function meta_filter(plugin_name, plugin_conf)
+local function meta_filter(route, plugin_name, plugin_conf)
```

Comment:
The `route` actually can be `route` or `global_rule`?

Reply:
How about using `api_ctx.conf_id` and `plugin_name` as union key.